src/ResourceManager/OperationalInsights/ChangeLog.md

```diff
@@ -21,6 +21,9 @@
 * Added Location Completer to -Location parameters allowing tab completion through valid Locations
 * Added ResourceGroup Completer to -ResourceGroup parameters allowing tab completion through resource groups in current subscription
 
+## Version 4.0.1
```

Comment:
Do not add the extra heading, please put this after the comments above, with no heading in between - changelogs versiosn are updated as part of the release.